.travis.yml

```diff
@@ -50,6 +50,14 @@ before_script:
   # Remember the Drupal installation path.
   - DRUPAL_DIR=$(pwd)/drupal
 
+  # Reference OG in the Drupal site.
+  - cp -R $MODULE_DIR $DRUPAL_DIR/modules/og
+
+  # Geting composer depdandecies.
+  - cd $DRUPAL_DIR
+  - composer require drush/drush:8.0.5
+  - vendor/drush/drush/drush dl composer_manager
```

Comment:
ok go ahead, you can still move it in composer.json when this is closer to completion :)